docs/changelog_de.md

```diff
@@ -12,6 +12,7 @@
 
 ### Story
 * Fix [#55](https://g1cp.org/issues/55) (aktualisiert): Grim erwähnt In Extremo im zweiten Kapitel nun auch wenn das Konzert noch nicht begonnen hat. Für weitere Informationen zum Fix, siehe [v1.1.0](https://g1cp.org/blob/master/docs/changelog_de.md#v110-01052021).
+* Fix [#128](https://g1cp.org/issues/128): Einer der Gardisten der Alten Mine hat nun nicht mehr zwei ENDE Dialogoptionen.
```

Comment:
Note: In the changelog entries of #42 and #244 there is a hyphen between ENDE and Dialogoptionen.